src/rest.js

```diff
@@ -166,20 +169,19 @@ function del(config, auth, className, objectId) {
         {
           objectId: objectId,
         },
-        options
+        options,
+        schemaController
       );
     })
     .then(() => {
       // Notify LiveQuery server if possible
-      config.database.loadSchema().then(schemaController => {
-        const perms = schemaController.getClassLevelPermissions(className);
-        config.liveQueryController.onAfterDelete(
-          className,
-          inflatedObject,
-          null,
-          perms
-        );
-      });
+      const perms = schemaController.getClassLevelPermissions(className);
```

Comment:
you sure ;)

Reply:
At this point the schemaData is loaded in the schemaController. The destroy call before it just deletes objects and the className is still in the schemaData